modules/API/API.js

```diff
@@ -1078,6 +1078,23 @@ class API {
         });
     }
 
+    /**
+     * Transfers the billingId from the parent localStorage.
+     *
+     * @returns {Promise<string>}
+     */
+    transferBillingId(): Promise<string> {
+        return new Promise(resolve => {
+            commands['transfer-billing-id'] = function(id) {
```

Comment:
Actually this mechanism is already implemented in the Transport class. Check https://github.com/jitsi/js-utils/blob/master/transport/Transport.js#L225. We already use this in external_api.js for example in the invite method. Maybe you can use it here as well?